lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -459,6 +460,15 @@ def non_fractional_currency?(currency)
       def emv_payment?(payment)
         payment.respond_to?(:emv?) && payment.emv?
       end
+
+      def error_code_from(response)
+        code = response['error']['code'] 
+        decline_code = response['error']['decline_code'] if code == 'card_declined'
```

Comment:
I don't think you want to do that actually, since `card_declined` just happens to be our mapping string as well as Stripe's. Stripe could change that string and this snippet would suggest they're supposed to be the same.